docs/vault.md

```diff
@@ -416,7 +416,7 @@ Connect to Vault using SSH
 ### Example
 
 ```bash
-sh1 vault ssh --vault my-vault
+h1 vault ssh --vault my-vault
```

Comment:
{{scope}} missing?

Reply:
No, it's builded documentation. See that fix https://github.com/hyperonecom/h1-cli/pull/137/files/a6b263b2785d9e8c3e77e6a03dde5fa3c3144d8e#diff-c7b006c46344ce3aabf1b9dfb58e75e5 .

Reply:
Upss, I miss read the location